library/Icinga/Web/Response.php

```diff
@@ -198,6 +205,30 @@ public function getRequest()
         return $this->request;
     }
 
+
+    /**
+     * Get whether to instruct client side script code to reload JS
+     *
+     * @return bool
+     */
+    public function isReloadJs()
+    {
+        return $this->reloadJs;
+    }
+
+    /**
+     * Set whether to instruct client side script code to reload JS
+     *
+     * @param   bool    $reloadJs
```

Comment:
Please remove the extra spaces from PHPDoc. I know we in indented everything in the past, but it is enough to add alignment around parameter names (if there is more than one), i.e.

```php
/**
 * Example doc
 *
 * @param int    $short    Something short
 * @param string $fullname Full name
 *
 * @return $this
 */
function foobar($short, $fullname)
{
    return $this;
}
```

Reply:
Should I fix this too in any other PHPDoc in this file?